src/client/hooks/useKeyEvent/index.tsx

```diff
@@ -7,13 +7,16 @@ import { blockTypes } from '/client/constants/tetriminos';
 import { keyboard } from '/client/constants/keyboard';
 import Block from '/client/models/block';
 
-export const useKeyEvent = ({ setIsPause, setMatrix, setBlock }
+export const useKeyEvent = ({ setIsPause, setMatrix, setBlock, isLeader }
   : {
     setIsPause: Dispatch<SetStateAction<boolean>>,
     setMatrix: Dispatch<SetStateAction<Matrix>>,
     setBlock: Dispatch<SetStateAction<Block>>,
+    isLeader?: boolean,
   }): void => {
+  console.log('Leader 1:', isLeader); // ok
   useKey((_key: number, { keyCode }: { keyCode: number }) => {
+    console.log('Leader 2:', isLeader); // TODO: prop is not visible here, how to pass it?
```

Comment:
Idk how to pass isLeader here (to handle Play key press conditionally)

Reply:
like this?
![image](https://user-images.githubusercontent.com/6814254/97073749-17706980-15e1-11eb-9efb-dde5b8171936.png)

Reply:
owh use setCurrentUser instead, and wrap this function with setCurrentUser
like this
![image](https://user-images.githubusercontent.com/6814254/97074420-4edf1600-15e1-11eb-9802-14cbe01c84f2.png)